internal/smb2/request.go

```diff
@@ -1050,7 +1067,7 @@ func (c *IoctlRequest) Encode(pkt []byte) {
 	le.PutUint32(req[4:8], c.CtlCode)
 	c.FileId.Encode(req[8:24])
 	le.PutUint32(req[32:36], c.MaxInputResponse)
-	le.PutUint32(req[36:40], c.OutputOffset)
+	//le.PutUint32(req[36:40], c.OutputOffset)
```

Comment:
**nitpick:** Since `offsetOutput` is set on L1080, consider simply deleting the code rather than commenting it out and/or noting that outputOffset will be added at a later point.